pkg/storaged/lvol-tabs.jsx

```diff
@@ -331,6 +350,10 @@ function lvol_shrink(client, lvol, info, to_fit) {
         if (vdo)
             shrink_size = vdo.physical_size + crypto_overhead;
 
+        const stratis_bdev = client.blocks_stratis_blockdev[content_path];
+        if (stratis_bdev)
+            shrink_size = Number(stratis_bdev.TotalPhysicalSize) + crypto_overhead;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230809-122740-f1602c20-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)